packages/onboarding/src/scroll-to-top/index.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useLocation } from 'react-router-dom';
+
+const ScrollToTop: React.FunctionComponent = () => {
```

Comment:
Seeing as we only scroll to top for [specific viewports in Gutenboarding](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx#L184) I wonder if it would be worth passing in viewports this would apply to (this would default to all viewports) and use the [useViewportMatch](https://github.com/Automattic/wp-calypso/blob/trunk/packages/domain-picker/src/domain-picker/suggestion-item.tsx#L64) hook

Reply:
Hmm, I agree that it's a bit of a scope creep. I've created an issue ([#48161](https://github.com/Automattic/wp-calypso/issues/48161) for this.